apps/right-to-rent-check/acceptance/features/tenancy-start.js

```diff
@@ -7,30 +7,36 @@ Feature('Tenancy Start Page');
 
 Before((
   I,
-  tenancyStartPage
+  propertyAddressPage
 ) => {
-  I.visitPage(tenancyStartPage, steps);
+  I.amOnPage(propertyAddressPage.url);
 });
 
 Scenario('The correct fields are on the page', (
   I,
   tenancyStartPage
 ) => {
+  I.completeToStep(`/${tenancyStartPage.url}`, {
+    'living-status': 'yes'});
   I.seeElements(_.values(tenancyStartPage.fields));
 });
 
 Scenario('I see an error if I submit without completing the fields', (
   I,
   tenancyStartPage
 ) => {
+  I.completeToStep(`/${tenancyStartPage.url}`, {
+    'living-status': 'yes'});
```

Comment:
Not exactly, I pass in another object in later tests,`'property-address-postcode'` [see the examples](https://github.com/UKHomeOffice/right-to-rent-check/pull/44/files/dab78352b58bb14dcb26758f17a037c036058d0f#diff-0de0e54bd9d066d8be9a823dd85bff4dR90)